static/app/components/replays/replayController.tsx

```diff
@@ -117,29 +119,50 @@ function ReplayCurrentTime() {
   );
 }
 
-function ReplayPlaybackSpeed({
-  speedOptions,
-  isCompact,
-}: {
-  isCompact: boolean;
-  speedOptions: number[];
-}) {
-  const {setSpeed, speed} = useReplayContext();
+function ReplayOptionsMenu({speedOptions}: {speedOptions: number[]}) {
+  const {setSpeed, speed, isSkippingInactive, toggleSkipInactive} = useReplayContext();
+  const SKIP_OPTION_VALUE = 'skip';
+
   return (
-    <CompactSelect
-      triggerProps={{
-        size: 'xs',
-        prefix: isCompact ? '' : t('Speed'),
-      }}
-      value={speed}
-      options={speedOptions.map(speedOption => ({
-        value: speedOption,
-        label: `${speedOption}x`,
-        disabled: speedOption === speed,
-      }))}
-      onChange={opt => {
-        setSpeed(opt.value);
-      }}
+    <CompositeSelect<SelectValue<string | number>>
+      placement="bottom"
```

Comment:
Do we need to specify this?

What happens when the video is in fullscreen, what direction does the dropdown take?

Reply:
So if I don't specify the placement when the sidebar is on the right side the dropdown menu gets cut off (the screenshot has the end of one screen into the next)

![Screen Shot 2022-07-14 at 1 21 56 PM](https://user-images.githubusercontent.com/3721977/179045080-2577954c-ab49-4f42-8bcd-b6f130970339.png)

I think it might even still get cut off a tiny bit as is when set to bottom where it's place more in the middle. We will likely have to set this depending on the layout we end up using (or dynamically based on a layout option). 

It opens upwards in full screen mode but I did notice a bug where the menu isn't activating on the first click but the second so I'm checking into that one now
![Screen Shot 2022-07-14 at 1 18 17 PM](https://user-images.githubusercontent.com/3721977/179045676-48ac5e71-fbea-4fc1-87a5-a419852ff3ad.png)